project-base/storefront/components/Blocks/PromoCode/PromoCode.tsx

```diff
@@ -62,7 +62,7 @@ export const PromoCode: FC = () => {
                                     applyPromoCodeToCart(promoCodeFormData.promoCode),
                                 )}
                             >
-                                <div className="flex gap-2 max-w-sm">
+                                <div className="flex max-w-sm">
```

Comment:
In this case, I would stick to the gap with the original design as well. Please check https://shopsys.atlassian.net/browse/SSP-2527 for more info.